sdk/src/client.ts

```diff
@@ -13,10 +13,11 @@ const defaultGlobals: ClientGlobals = {
 /**
  * Creates a client to communicate with a Directus app.
  *
- * @param url The URL to the Directus app.
- * @param config The optional configuration.
+ * @param {string} url - The URL to the Directus app.
+ * @param {ClientOptions} [options] - The client options. Defaults to defaultGlobals.
+ *     {@see {@link ClientOptions}} and {@see {@link ClientGlobals}}
```

Comment:
We don't need types in JSDoc as they are already provided by TypeScript.

```suggestion
 * @param url The URL to the Directus app.
 * @param options The client options. Defaults to the standard implementation of `globals`.
```